script/7_deployWdivETHContract.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+echo "Instructing the shell to exit immediately if any command returns a non-zero exit status..."
+set -e
+echo "Done."
+
+echo "Navigating to the root directory of the project..."
+cd ../
+echo "Done."
+
+echo "Setting environment variables..."
+source .env
+echo "Done."
+
+echo "Deploying OptimismMintableERC20 wdivETH token smart contract..."
+cast send --rpc-url $L2_RPC_URL "0x4200000000000000000000000000000000000012" "createOptimismMintableERC20WithDecimals(address,string,string,uint8)" $L1_TOKEN_ADDR_DIVA "Wrapped Diva Ether Token" wdivETH 18 -i
```

Comment:
Should `wdivETH` also be inside double quotes because it's a string like  "Wrapped Diva Ether Token"?

`-i`: why it needs to be in interactive mode? We have a deployer private key inside `.env` file.

Reply:
I think the double quotes are irrelevant, I will test adding them as well. It does not need to be interactive but I wanted to have the same flow as the USDT token, no preference from my side

Reply:
I've overlooked `-i` when I was reviewing USDT token deployment script.
I was quite surprised what to do when we're deploying it and I observed script requires input from user.

Would be better to do like it's in other scripts where deployer private key is taken from `.env` file.

Reply:
Sounds good, should we also open a PR to change the USDT deployment as well?

Reply:
As you prefer.

I think we won't use USDT deployment script for `Mainnet` deployment anymore, right?
But for the (potential) future use, we may change it to be more aligned with other deployment scripts.

Reply:
BTW, how do you handle it in other deployment scripts? I see that there is no private key specified there

Reply:
Inside deployment script we import private key from `.env` file.
It's the same in all deployments scripts.
Example:
https://github.com/LiskHQ/lisk-contracts/blob/development/script/contracts/L2/L2Airdrop.s.sol#L20-#L21